.github/workflows/piplock-renewal.yaml

```diff
@@ -3,9 +3,9 @@
 name: Pipfile.locks Renewal Action
 
 on:  # yamllint disable-line rule:truthy
-  # Triggers the workflow every Monday at 22pm UTC am 0 22 * * 1
+  # Triggers the workflow every Wednesday at 1am UTC
```

Comment:
Maybe slightly earlier time so that at least America has night would be better, but that doesn't really matter in the end.